### ISSUE TITLE
Shutdown Events should not occur if SPMC is not the active app

### ISSUE THREAD

**Issue by elmerohueso** [koying/SPMC#16]
If Shutdown Timer is set, shutdown event will run even if SPMC is not active (ie. device will shutdown while watching Netflix).  I'm thinking this might be prevented by resetting the shutdown timer (in CApplication::CheckShutdown()) if !m_AppFocused.

I'd test, but I can't get SPMC dependencies to build for some reason (XBMC/Kodi builds fine, but it doesn't have all the Android power management stuff needing for a good test).

**Comment by koying** [comment]
Is this SPMC or even Android-specific?
I mean wouldn't XBMC act the same on, e.g., windows?

**Comment by elmerohueso** [comment]
It's not SPMC-specific (I brought it up over on the XBMC forums but got no response).  It's surprisingly not Android-specific, either.  However, it might have more of an impact in SPMC as it runs as a Launcher (thus, is always running in the background).

I did try resetting the shutdown timer (in CApplication::CheckShutdown()) if !m_AppFocused, but that doesn't seem to work.  I tried tinkering a bit more but wasn't able to figure it out before I got sidetracked by other stuff.  I'm hoping you might have a better idea on how to implement this.
